README.md

```diff
@@ -108,7 +108,10 @@ kube-burner-ocp cluster-density-v2 --iterations=1 --churn-duration=2m0s --churn-
 
 `.TOKEN` can be captured by running `TOKEN=$(oc create token -n openshift-monitoring prometheus-k8s)`
 
-!!! Note 
+!!! Note
+
+    Avoid passing absolute path of the file with --metrics-endpoint option
```

Comment:
What was the issue with passing an absolute path?

Reply:
I was getting below error when passing absolute path

time="2024-08-14 18:24:16" level=fatal msg="Error reading metricsEndpoint /root/work/e2e-benchmarking/workloads/kube-burner-ocp-wrapper/metrics-profiles/metrics-endpoint.yaml: Get \"/root/work/e2e-benchmarking/workloads/kube-burner-ocp-wrapper/metrics-profiles/metrics-endpoint.yaml\": unsupported protocol scheme \"\"" file="utils.go:32"

Reply:
Thanks for making this change. This has been an open issue for a while. This [part](https://github.com/kube-burner/kube-burner/blob/main/pkg/util/url_reader.go#L33) of code was supposed to gracefully handle the error that we are seeing here. Let me get this change in and have a follow up story to further investigate and fix this.

Reply:
Follow up story: https://github.com/kube-burner/kube-burner/issues/683